examples/pytorch_imagenet_resnet50.py

```diff
@@ -40,6 +41,10 @@
                     help='SGD momentum')
 parser.add_argument('--wd', type=float, default=0.00005,
                     help='weight decay')
+parser.add_argument('--batches-per-allreduce', type=int, default=1,
```

Comment:
nit: let's bring this up after `--fp16-allreduce`, since this help section is for model parameters, while `--batches-per-allreduce` is more of performance optimization parameter.